project/src/pages/add-review-screen.tsx

```diff
@@ -1,166 +1,46 @@
 import Breadcrumbs from '../components/breadcrumbs/breadcrumbs';
 import Header from '../components/header/header';
+import CommentForm from '../components/comment-form/comment-form';
+import { AppRoute } from '../const';
+import { useParams } from 'react-router-dom';
+import { Films, Film } from '../types/films';
+
+type AddReviewScreenProps = {
+  films: Films
+};
+
+function AddReviewScreen({ films }: AddReviewScreenProps): JSX.Element {
+  const { id }: {id: string}= useParams();
```

Comment:
мне кажется так понагляднее будет
```suggestion
  const { id } = useParams<PageParams>();
```